### PR TITLE
(Bug) Dutch- Publish page: uploaded file'Auth-os_addresses.txt' contains wrong names of contracts

### LINKED ISSUE #1101: (Bug) Dutch- Publish page: uploaded file'Auth-os_addresses.txt' contains wrong names of contracts
Expected: 
 - Auth-os DutchIdx address
 - Auth-os Dutch Crowdsale address
 - Auth-os Dutch Token address
Actual result:
 - Auth-os MintedCappedIdx address
 - Auth-os Sale address
 - Auth-os Token address
  

If you are reporting a problem with Token Wizard, please include the following information:

### Which network did you use? (Mainnet, Kovan, Rinkeby, etc.)

*POA core*

### If you were able to create it, what is the URL of your crowdsale?

*Yhttp://localhost:3000/contribute?addr=0x1ab8f02b197fce9d149db2f015ac7ab22f2cfad2&networkID=99*

### Do you have screenshots showing the problem?
screen shot 2018-08-18 at 5 26 06 am

### Do you see errors in the dev console? If yes, please include a screenshot
no

---

### PR DESCRIPTION
**Pull request by mariano-aguero**
Closes #1101 

- Check strategy of the crowdsale
- Create labels regarding selected strategy

### CODE REVIEW

**Comment by dennis00010011b** [comment]
@mariano-aguero 
For avoid to create one more PR it could be nice to make some additional work in this PR:
 add to `Auth-os_addresses.txt` for Minted crowdsale two contract’s adresses:
   1. Sale_manager ( var REACT_APP_MINTED_CAPPED_CROWDSALE_MANAGER_ADDRESS)
2. Token manager ( var REACT_APP_MINTED_CAPPED_TOKEN_MANAGER_ADDRESS)

**Comment by mariano-aguero** [comment]
@dennis00010011b 

Can you specific de ID of the issue ?

**Comment by dennis00010011b** [comment]
@mariano-aguero 
Sorry! https://github.com/poanetwork/token-wizard/issues/1118

**Comment by mariano-aguero** [comment]
@dennis00010011b 

> For avoid to create one more PR it could be nice to make some additional work in this PR:
add to Auth-os_addresses.txt for Minted crowdsale two contract’s adresses:

Done

Closes #1118

**Comment by dennis00010011b** [comment]
@mariano-aguero 
Dutch-Publish page: app crashed while attempting to download file with metadata
Steps for reproducing:  create Dutch crowdsale with any valid parameters  and proceed until Publish page.
Issue not reproducible for Minted
screen shot 2018-09-02 at 5 43 33 am
[dutch_download_crach.log](https://github.com/poanetwork/token-wizard/files/2343322/dutch_download_crach.log)

File: src/components/stepFour/utils.js [hunk 1]
```diff
@@ -1008,6 +1008,65 @@ export const SUMMARY_FILE_CONTENTS = networkID => {
     return []
   }
 
+  const isDutchStrategy = crowdsaleStore.strategy === CROWDSALE_STRATEGIES.DUTCH_AUCTION
+  const labelIdx = isDutchStrategy ? 'DutchIdx' : 'MintedCappedIdx'
+  const labelCrowdsale = isDutchStrategy ? 'Dutch Crowdsale' : 'Sale'
+  const labelToken = isDutchStrategy ? 'Dutch Token' : 'Token'
+  const labelSaleManager = isDutchStrategy ? 'Dutch Crowdsale manager' : 'Sale manager'
```
Comment (vbaranov) [inline src/components/stepFour/utils.js]: As Dutch strategy has no managers smart-contracts, here and in the next line, we could remove conditional operator and just leave values for Minted capped strategy

Comment (mariano-aguero) [reply src/components/stepFour/utils.js]: Change done

**Pull request merged**